sdks/java/io/hadoop-format/build.gradle

```diff
@@ -96,7 +96,6 @@ dependencies {
   testCompile library.java.hamcrest_library
   testCompile library.java.testcontainers_postgresql
   testCompile library.java.netty_all
-  testRuntimeOnly library.java.log4j_core
```

Comment:
It seems adding log4j-api to the library was the root cause of this issue. This forced dependency resolution to go to 2.14.1 for the runtime dependency in GrpcVendoring, which was reflected in any downstream shaded jars (e.g. the SDK harness).

Reply:
Oh interesting and tricky